tempy/widgets.py

```diff
@@ -181,11 +182,78 @@ def make_footer(self, footer):
         self._make_table_part('footer', footer)
 
     def make_caption(self, caption):
-        """Adds/Sobstitute the table's caption."""
+        """Adds/Substitutes the table's caption."""
         if not hasattr(self, 'caption'):
             self(caption=Caption())
         return self.caption.empty()(caption)
 
+    def col_class(self, css_class, col_index=None):
+        # adds css_class to every cell
+        if col_index is None:
+            for row in self.body.childs:
+                for cell in row.childs:
+                    cell.attr(klass=css_class)
+        elif col_index >= 0:
+            for row in self.body.childs:
+                if col_index < len(row.childs):
+                    row.childs[col_index].attr(klass=css_class)
```

Comment:
At the end of this method, if no cell is affected (i.e: every row is shorter that the given col_index) i'd raise a WidgetError/IndexError